### ISSUE TITLE
potential video preprocessing on DiDeMo videos

### ISSUE THREAD

**Issue by RenShuhuai-Andy** [klauscc/VindLU#7]
Thanks for the great work!

I used your released checkpoint `25M-retrieval-didemo.pth` and directly conducted inference, but got accuracy lower than that reported in the repo:

reported (12 test frm): 61.2
re-inference (12 test frm): 59.5
re-inference (32 test frm): 59.8

I noticed that it seems you prepross the videos in DiDeMo, i.e., `2fps_360_trimed30` in the following config. I wonder if you resize the frames to 360 and what does `trimed30` mean? Can you provide the preprossing script if it has great impact to the final results? Thanks a lot.

https://github.com/klauscc/VindLU/blob/30465487e8314fa1df45b1457b0313f25649e054/configs/ret_didemo.py#L5-L9

**Comment by klauscc** [comment]
Hi,
I just tested the checkpoint and got exactly the same results as reported (see [evaluation log](https://drive.google.com/file/d/1RtarNr-1RmnoB7TjaA-iznbraA7nZkz1/view?usp=share_link)). So the issue may be on the data side.

We resized the videos to 2fps with short side of 360.  `trim30` means we only keep the first 30 seconds of each video. The original DiDeMo paper, CLIPBert, Singularity, LAVENDER also uses the first 30 seconds of the video.
